src/main/java/core/basesyntax/strategy/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+
+    @Override
+    public String getReport(Storage storage) {
+        StringBuilder report = new StringBuilder();
+        report.append("fruit,quantity\n");
```

Comment:
move literals and separators to the constant fields. Also better use System.lineseparator()